src/printing/tests.rs

```diff
@@ -0,0 +1,82 @@
+use proc_macro2::{Literal, TokenTree};
+use quote::ToTokens;
+
+use crate::{ByteLit, ByteStringLit, CharLit, FloatLit, IntegerLit, StringLit};
+
+#[test]
+fn it_preserves_bool_true() {
+    let other = crate::BoolLit::True
+        .to_token_stream()
+        .into_iter()
+        .next()
+        .and_then(|v| {
+            if let TokenTree::Ident(v) = v {
+                Some(v)
+            } else {
+                None
+            }
+        })
+        .unwrap();
+    assert_eq!(other, "true")
```

Comment:
```suggestion
    let v = crate::BoolLit::True.to_token_stream().into_iter().collect::<Vec<_>>();
    assert_eq!(v.len(), 1);
    let TokenTree::Ident(ident) = v[0] else { panic!("not an ident") };
    assert_eq!(ident, "true");
```

How about this? This also checks that there are no extra token trees in the stream.